services/algorithms/__init__.py

```diff
@@ -14,6 +14,9 @@ def __init__(self, **kwargs):
     self.loss_function = getattr(keras.losses, kwargs[LOSS_FUNCTION])
     self.learning_rate = float(kwargs[LEARNING_RATE])
     self.optimizer = getattr(keras.optimizers, kwargs[OPTIMIZER])(lr=self.learning_rate, epsilon=1.5e-4) #TODO: this epsilon only works for Adam
+    self.clip_norm = float(kwargs[CLIP_NORM])
+    # Does not perform clipping unless clip_norm is set to non-zero float
+    if self.clip_norm == 0: self.clip_norm = None
```

Comment:
This line isn't necessary for your code to work.  In Python 0.0 == False and 0 == False so your existence check down the line will work either way.

Regardless, let's make the value in default_json "null" and make line 17:
`self.clip_norm = float(kwargs.get(CLIP_NORM)) if kwargs.get(CLIP_NORM) else None`